app/models/validations/sales/financial_validations.rb

```diff
@@ -44,4 +69,14 @@ def validate_percentage_owned_not_too_much_if_older_person(record)
       record.errors.add :type, I18n.t("validations.financial.staircasing.older_person_percentage_owned_maximum_75")
     end
   end
+
+private
+
+  def is_relationship_child?(relationship)
```

Comment:
there are some similar methods in validations/household_validations.rb - can we put these all into shared_validations?